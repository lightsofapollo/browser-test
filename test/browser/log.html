<!DOCTYPE HTML>
<html>
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <title>body</title>
</head>
<body>
<script src="/browser_test/browser_test.js" type="text/javascript" charset="utf-8"></script>
<script type="text/javascript" charset="utf-8">
browserTest.createSuite(2);
console.log('I CREATE THE LOG');

browserTest.createTest('start one', new Date());
browserTest.passTest();

browserTest.finishSuite(new Date());
</script>
</body>
</html>

