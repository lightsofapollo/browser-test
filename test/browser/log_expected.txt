1..2
# LOG: I CREATE THE LOG - (http://localhost:54308/test/browser/log.html?url=ws%3A%2F%2Flocalhost%3A54308%2Ftest:11:1)
ok 1 start one
# PASSED: 1
# PENDING: 0
# FAILED: 0
